application/src/main/java/org/opentripplanner/transit/model/basic/Money.java

```diff
@@ -147,6 +147,15 @@ public Money half() {
     return new Money(currency, IntUtils.round(amount / 2f));
   }
 
+  /**
+   * Returns the instance rounded down to the nearest multiple of 5 cents
+   * So $0.25 becomes $0.10
```

Comment:
25 cents become 10 cents!? That doesn't sound right.

Reply:
I had in my head how we use half() first.